integr8ly/integreatly-operator
# Prepare release script update

# Description
remove redis and openshift ci from container images list in RHOAM CSV.

# Verification 
- run `OLM_TYPE=managed-api-service SEMVER=5.0.0 make release/prepare` and confirm that the `redis-32-rhel7`, `mysql-57-rhel7` and `openshift-cli` container images aren't listed anymore in the metadata.annotations.containerImages of RHOAM 5.0.0 CSV
- confirm all other container images are present (https://gist.github.com/MStokluska/eb387962b369120e1a7cd4ac367f03f9)
